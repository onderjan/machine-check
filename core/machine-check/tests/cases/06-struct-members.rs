#[machine_check_macros::machine_description]
mod machine {
    struct A;

    struct B();

    struct C(::machine_check::Bitvector<1>);

    struct D {
        d: ::machine_check::Bitvector<1>,
    }
}

fn main() {}
