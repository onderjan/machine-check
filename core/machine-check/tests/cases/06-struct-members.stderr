error: machine-check: Struct without named fields not supported
 --> tests/cases/06-struct-members.rs:3:5
  |
3 |     struct A;
  |     ^^^^^^^^^

error: machine-check: Struct without named fields not supported
 --> tests/cases/06-struct-members.rs:5:5
  |
5 |     struct B();
  |     ^^^^^^^^^^^

error: machine-check: Struct without named fields not supported
 --> tests/cases/06-struct-members.rs:7:5
  |
7 |     struct C(::machine_check::Bitvector<1>);
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
